mlflow/langchain/__init__.py

```diff
@@ -285,13 +291,22 @@ def log_model(
     """
     import langchain
 
-    if type(lc_model) != langchain.chains.llm.LLMChain:
+    if (
+        type(lc_model) != langchain.chains.llm.LLMChain
+        and type(lc_model) != langchain.agents.agent.AgentExecutor
+    ):
         raise TypeError(
```

Comment:
Nit (especially nitty since this isn't your code, sorry): This should be `MlflowException` with error code `INVALID_PARAMETER_VALUE`